projects/1.20/assets/my-nethers-delight/mynethersdelight/lang/zh_cn.json

```
@@ -0,0 +1,87 @@
+{
+  "block.mynethersdelight.block_of_powdery_cannon": "烟药竹块",
+  "block.mynethersdelight.block_of_stripped_powdery_cannon": "去皮烟药竹块",
+  "block.mynethersdelight.bullet_pepper": "烟药弹椒",

```

```suggestion
  "block.mynethersdelight.bullet_pepper": "爆弹椒",
```
原文 Bullet Pepper 不含 Powdery 前缀，译名中无需保留“烟药”前缀，且“烟药弹椒”四个字不是很顺口。在保持和烟药竹的相关性的前提下，可译为“烟弹椒”。
但从纹理角度上看，其实 Bullet Pepper 作为果实和烟药竹的整体外观实际上差异较大，译名完全放弃和烟药竹的相关性也未尝不可（先例：橡树和苹果），根据其“一触即爆”的描述，建议译为“爆弹椒”。